### PR TITLE
[WIP] APPDUX-407 Lays out components for new realm form

### PR DESCRIPTION
**Pull request by srambach**
Lay out the components for the New Realm form.

APPDUX-407, also issue #12

### CODE REVIEW

**Comment by ssilvert** [comment]
I don't think App.tsx is the right place to put this.   

I suspect that the work @dlabaj is doing with routing will include a way to implement putting the form in the content area as shown in https://marvelapp.com/prototype/ei9eebh/screen/71477794 ?

For now, storybook might be the right way to develop this because we don't yet have a way to display it in the application.

**Comment by ssilvert** [comment]
> Looking good @srambach ... add in the divider and this should be all set for the layout! Looks like we have some issues getting the correct fonts to load through `/assets`, which the build is complaining about in the console messages. Is this a snowpack config issue @edewit @ssilvert? I'm happy to look into it!

We had the same issue in account console.  I know how to fix it.  Added issue #20

**Comment by jenny-s51** [comment]
@ssilvert awesome, thank you for taking on the assets issue -  I'm curious to see your fix! @dlabaj  and I were just discussing `react-router` ... once he adds that package I was planning on tying it into the dropdown via my realm selector issue and routing to Sarah's new page. I do agree that we should revert `App.tsx` back to what it was originally though

**Comment by srambach** [comment]
@ssilvert Definitely was not my intention to leave it there in `App.tsx`. However, I do think that the `<PageSection>` might need to come out of `App.tsx`. The content area will likely nearly always have multiple page sections, and the additional one wrapping them causes added padding. We could remove the padding, but I don't know of a compelling reason to have that element there.
Storybook might be a good workflow for us; I'm interested in how we can make that work.

File: src/forms/realm/NewRealmForm.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+type NewRealmFormProps = {
+  realm: string;
+};
+
+export const NewRealmForm = ({ realm }: NewRealmFormProps) => {
+  return (
+    <>
+      <PageSection variant="light">
+        <TextContent>
```
[the remark this answers is not in the document]
Comment (srambach) [reply src/forms/realm/NewRealmForm.tsx:25]: 👍 thanks, good catch

**Comment by ssilvert** [comment]
> @ssilvert awesome, thank you for taking on the assets issue - I'm curious to see your fix! 
Actually, I reassigned to Erik.  This stuff has changed with snowpack 2 so it wouldn't be a quick fix for me.

File: src/forms/realm/NewRealmForm.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            <TextInput
+              isRequired
+              type="text"
+              id="kc-realm-name"
+              name="kc-realm-name"
+              // value={value2}
+              // onChange={this.handleTextInputChange2}
+            />
+          </FormGroup>
+          <FormGroup label="Enabled" fieldID="kc-realm-enabled-switch">
```
Comment (edewit) [inline src/forms/realm/NewRealmForm.tsx:54]:
```suggestion
          <FormGroup label="Enabled" fieldId="kc-realm-enabled-switch">
```

**Comment by edewit** [comment]
fonts problem solved with #21

**Pull request merged**